easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -806,19 +811,8 @@ def _parse_dependency(self, dep, hidden=False, build_only=False):
         tc = copy.deepcopy(self['toolchain'])
         tc_spec = dependency['toolchain']
         if tc_spec is None:
-            if build_option('minimal_toolchains'):
-                self.log.debug("Looking for minimal toolchain for dependency %s (parent toolchain: %s)...",
-                               dependency, tc)
-                # update the toolchain with the minimal value
-                orig_tc = tc
-                tc = robot_find_minimal_toolchain_of_dependency(dependency, self.modules_tool, parent_tc=tc)
-                if tc is None:
-                    raise EasyBuildError("No easyconfig for %s that matches toolchain hierarchy generated by %s",
-                                         dependency, orig_tc)
-                else:
-                    self.log.debug("Obtained minimal toolchain: %s", tc)
-            else:
-                self.log.debug("Inheriting parent toolchain %s for dependency %s", tc, dependency)
+            self.log.debug("Inheriting parent toolchain %s for dependency %s (for now)", tc, dependency)
```

Comment:
With `minimal-toolchains` you might be replacing this toolchain with one further down the hierarchy when the dependencies are finalised. Probably could do with replacing `(for now)` by `(until dependencies are finalised)`